lib/Frontend/CachingUtils.cpp

```diff
@@ -472,16 +472,10 @@ static Expected<ObjectRef> mergeCASFileSystem(ObjectStore &CAS,
 
 Expected<IntrusiveRefCntPtr<vfs::FileSystem>>
 createCASFileSystem(ObjectStore &CAS, ArrayRef<std::string> FSRoots,
-                    ArrayRef<std::string> IncludeTrees,
-                    ArrayRef<std::string> IncludeTreeFileList) {
-  assert(!FSRoots.empty() || !IncludeTrees.empty() ||
+                    const std::string &IncludeTree,
```

Comment:
Shouldn't we prefer `StringRef` here?

Reply:
The input is always a `std::string` from FrontendOptions. I don't see StringRef is necessary and it adds overhead if not inlined.